### DISCUSSION TITLE
Issues with "moderen" gTLD's

### DISCUSSION THREAD

**Discussion by RamonSmit** [postalserver/postal#2144] [Help with using Postal]
Hello everyone,

I'm having some difficulties with a domain name that keeps messages on "pending" in Postal. It's a domain name with the .quest gTLD. 

Steps i've taken: 
- DNS verified on September 07, 2022 10:38
- Credential "Process all messages" 
- Mails from other domain names with ".nl", ".com", ".email" go trough. 
- No logs found 

For what I can see, it looks like the messages are not picked up by "UnqueueMessageJob"

Does anyone else found a similar issue? 

Thanks in advance.

**Comment by willpower232** [comment]
I highly doubt the tld is related to whatever problem you're experiencing unless your server is having a really bad time trying to connect to wherever the messages are intended to be delivered. It might be worth enabling sentry to track issues within your Postal server in case the logs aren't enough.

**Comment by RamonSmit** [comment]
I can try Sentry indeed. It's only for the .quest gTLD, the queue worker is not even trying to process them. All other mails get processed except the .quest in this case. It's continue hanging on "pending", not even failing. 

I will try Sentry.

**Comment by RamonSmit** [comment]
I've enabled Sentry and there are no errors logged in Sentry. I've inspected rabbitmq and I'm not seeing anything weird. 

Only thing that I'm seeing is that the messages are stuck in `queued_messages` 

Any suggestions?

Edit: 

When I disable IP pools, the system starts queueing again. Might me something funny with IP pools?

**Reply by willpower232** [reply]
I tried using example.quest which doesn't exist but our Postal still tried

![image](https://user-images.githubusercontent.com/1619102/194616433-e7217452-26f5-4dcd-bd8d-ca3eaf02dbaa.png)

Unfortunately we don't use IP Pools so I can't confirm anything about that.

**Reply by RamonSmit** [reply]
Thanks for responding. I've looked into it with a friend of mine. It might be related to #1190
